openedx/features/course_experience/templates/course_experience/course-outline-fragment.html

```diff
@@ -176,7 +176,21 @@ <h4 class="subsection-title">
         </ol>
     % endif
 </main>
+<script>
+    $(document).ready(function(){
+        var url_hash = window.location.hash;
 
+        if(url_hash !== ""){
+            var block = document.getElementById(url_hash);
+            if(block.classList.contains('subsection-text')){
+                var parent_li = block.parentElement.parentElement.parentElement;
+                var parent_block = parent_li.children[0];
+                parent_block.click()
```

Comment:
This seems like a bit much. As mentioned elsewhere, perhaps you can add markup to make this cleaner and more reliable.